CHANGELOG.md

```diff
@@ -1,5 +1,29 @@
 # Changelog
 
+## Python Version 1.1.6 (2023-18-11)
+
+## NPM Version 1.1.1 (2023-18-11)
+
+### Features
+
+- **python-sdk:** Created model parser extension for Google’s Gemini ([#478](https://github.com/lastmile-ai/aiconfig/pull/478), [cookbook](https://github.com/lastmile-ai/aiconfig/pull/483))
+- Added attachment field to PromptInput schema ([#473](https://github.com/lastmile-ai/aiconfig/pull/473))
```

Comment:
it would be good to say **what** this enables. In this case, probably "enables multiple inputs of any MIME type for the same prompt, to support multi-modal models like GPT-4V and Gemini"